contracts/services/ComponentOwnerService.sol

```diff
@@ -2,6 +2,7 @@
 pragma solidity ^0.8.0;
 
 import "../modules/ComponentController.sol";
+import "../modules/PoolController.sol";

```

Comment:
i know we discussed it this way. in principle i don't much like that we now introduce a dependency to pool from component, ideally component wouldn't know about pool ... however, in absence of any alternative idea we probably just go with it liket it is

Reply:
i will leave a TODO here for the future.